test/distributed/test_c10d.py

```diff
@@ -3870,6 +3895,11 @@ def test_ddp_comm_hook_allreduce_hook_nccl(self):
     def test_default_ddp_comm_hooks_nccl(self):
         self._test_default_ddp_comm_hooks_nccl()
 
+    @requires_nccl()
```

Comment:
Any advice on how to test my changes? 
Trying to run these locally fails, because I don't have a CUDA-capable GPU.

Reply:
Need to reserve a machine with GPU.